dashboard/src/lib/date.ts

```diff
@@ -0,0 +1,14 @@
+import { differenceInDays } from 'date-fns';
+
+const RELATIVE_DAYS_EDGE = 3;
+
+export const shouldShowRelativeDate = (
+  date: string,
+  limitInDays: number = RELATIVE_DAYS_EDGE,
+): boolean => {
+  const treeDate = new Date(date);
+  const currentDate = new Date();
+  const days = differenceInDays(currentDate, treeDate);
+
+  return days <= limitInDays;
```

Comment:
```suggestion
  return days < limitInDays;
```

some rows are showing the label `4 days ago` (we define 3 days as maximum)